doc/templates/index.html

```diff
@@ -228,7 +228,7 @@ <h4 class="sk-landing-call-header">Who uses scikit-learn?</h4>
             </div>
             <div class="carousel-item">
             <img class="d-block mx-auto sk-who-uses-carousel-img img-thumbnail" src="_images/aweber.png" alt="aweber">
-            <em>"It allows us to do AWesome stuff we would not otherwise accomplish"</em>
+            <em>"It allows us to do Awesome stuff we would not otherwise accomplish"</em>
```

Comment:
I think strictly speaking this should be `awesome` (lower case A as well). Given the name of the company that the testimonial comes from (`AWeber`) I wonder if this typo (`AWesome`) is on purpose. However if no one can remember I'd vote for sticking to normal spelling rules and not have cute misspellings like this

Reply:
Thank you for your review and guidance.